src/commands/move.ts

```diff
@@ -138,15 +138,30 @@ function moveDown(editor: vscode.TextEditor, expand: boolean, mult: number) {
   }
 }
 
+/**
+ * Changes one character selections to be the direction specified. Return others unchanged.
+ * @param selection the original selection. Will be reversed if needed
```

Comment:
Nit: Capitalize "Selection" and "Direction" below. Add empty line between the summary and the `@param`s. Add period at the end of the line.

Reply:
Sorry but I cannot capitalize the `Selection` in `@param selection` -- it should be exactly the same as the parameter name based on how jsdoc works. But I've capitalized the descriptions for both parameters and made them full sentences because I think that's what you really meant.